internal/domain/sos_post/view.go

```diff
@@ -97,12 +93,16 @@ type UpdateSosPostView struct {
 	Conditions   []ConditionView   `json:"conditions"`
 	Pets         []pet.PetView     `json:"pets"`
 	Reward       string            `json:"reward"`
-	DateStartAt  string            `json:"date_start_at"`
-	DateEndAt    string            `json:"date_end_at"`
+	Dates        []SosDateView     `json:"dates"`
 	CareType     CareType          `json:"care_type"`
 	CarerGender  CarerGender       `json:"carer_gender"`
 	RewardAmount RewardAmount      `json:"reward_amount"`
 	ThumbnailID  int               `json:"thumbnail_id"`
 	CreatedAt    time.Time         `json:"created_at"`
 	UpdatedAt    time.Time         `json:"updated_at"`
 }
+
+type SosDateView struct {
+	DateStartAt string `field:"date_start_at"`
+	DateEndAt   string `field:"date_end_at"`
+}
```

Comment:
이제부터는 camelCase로 통일해볼까요?
```suggestion
type SosDateView struct {
	DateStartAt string `field:"dateStartAt"`
	DateEndAt   string `field:"dateEndAt"`
}
```

Reply:
좋습니당